geode-protobuf/src/main/java/org/apache/geode/internal/protocol/security/NoOpAuthenticator.java

```diff
@@ -12,23 +12,23 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.internal.protocol.protobuf.security;
+package org.apache.geode.internal.protocol.security;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 
 import org.apache.geode.internal.security.SecurityService;
+import org.apache.geode.security.AuthenticationFailedException;
 
 /**
  * An implementation of {@link Authenticator} that doesn't use its parameters and always returns
  * true.
  */
-public class NoOpAuthenticator implements Authenticator {
+public class NoOpAuthenticator implements Authenticator<Object, Object> {
+
   @Override
-  public Object authenticate(InputStream inputStream, OutputStream outputStream,
-      SecurityService securityService) throws IOException {
-    // this method needs to do nothing as it is a pass-through implementation
+  public Object authenticate(Object inputObject) throws AuthenticationFailedException {
     return new Object();
```

Comment:
Should we just return the same dummy object for each call rather than creating a new object each time.

Reply:
maybe just `null`?

Reply:
`null` returned